openssl-sys/build.rs

```diff
@@ -41,14 +39,15 @@ fn main() {
 }
 
 fn build_old_openssl_shim(is_old: bool) {
-        let mut config: gcc::Config = Default::default();
-        if is_old {
-            config.definitions.push(("OLD_OPENSSL".to_string(), None));
-        }
+    let mut config: gcc::Config = Default::default();
+    if is_old {
+        config.definitions.push(("OLD_OPENSSL".to_string(), None));
```

Comment:
This needs to be built in all cases because it provides the `HMAC_Init_ex_shim`, `HMAC_Update_shim` and `HMAC_Final_shim` functions that the main crate links to.